bin/setup_parity

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p parity
+echo > ./parity/passfile
+
+curl https://releases.parity.io/ethereum/v1.10.7/x86_64-unknown-linux-gnu/parity --output ./parity/parity
```

Comment:
we may want to update parity (open-ethereum as it's called now) or migrate to another client. what do you think?

Reply:
@ayrat555 I was thinking the same thing ;) I wanted to start with `parity` because that's what's currently used in CI.

How do you feed about `setup.parity`, `setup.openethereum`, `setup.geth`?

Reply:
I feel like we should be testing against all clients if we can. Definitely should have `geth`

Reply:
🤔 but the API is the same for all clients.

Reply:
It "should" be the same... But my experience has been that regressions get introduced from time to time.